plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/services/codemodernizer/CodeModernizerStartupActivity.kt

```diff
@@ -14,6 +14,7 @@ class CodeModernizerStartupActivity : StartupActivity.DumbAware {
      */
     override fun runActivity(project: Project) {
         if (!isCodeModernizerAvailable(project)) return
-        CodeModernizerManager.getInstance(project).tryResumeJob()
+        val codeModernizerInstance = CodeModernizerManager.getInstance(project)
+        codeModernizerInstance.tryResumeJob(true)
```

Comment:
Instead of moving the validation check to onstartactivity, we pass in a parameter here so we can keep methods private and scoped within codeModernizer class. Other calls to `tryResumeJob` will not have this parameter.